web_poet/pages.py

```diff
@@ -137,3 +150,20 @@ class WebPage(ItemPage[ItemT], ResponseShortcutsMixin):
 
 
 ItemWebPage = _create_deprecated_class("ItemWebPage", WebPage, warn_once=False)
+
+
+@attr.s(auto_attribs=True)
+class SelectorExtractor(Extractor[ItemT]):
```

Comment:
We should have it documented somewhere; I think only having reference API docs and not mentioning it elsewhere is not enough.